src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisplayNameProvider.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Microsoft.PowerFx.Core.UtilityDataStructures;
+
+namespace Microsoft.PowerFx.Core
+{
+    internal class DisplayNameProvider
+    {
+        // First is Logical Name, Second is Display Name
+        private BidirectionalDictionary<string, string> _displayNames;
```

Comment:
>string [](http://example.com/codeflow?start=40&length=6)

Should either of these be DName?

Reply:
They can both be, no harm in changing that.